<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMag</class>
 <widget class="QWidget" name="PageMag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QTableWidget" name="TabParams">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>选中</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>端一</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>端二</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>上限</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>波形</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="LayoutCtrl">
     <item row="0" column="9">
      <widget class="QPushButton" name="BtnSampleMag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>采集波形</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QComboBox" name="BoxStation">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>44</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>左工位</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>右工位</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="BoxMain">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="TextDebug">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QSpinBox" name="BoxAuxiliary">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QPushButton" name="BtnExitMag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>44</height>
        </size>
       </property>
       <property name="text">
        <string>保存退出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="TextAuxiliary">
       <property name="text">
        <string>副相：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="TextDir">
       <property name="text">
        <string>磁旋:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="BoxDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>不测</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>正转</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>反转</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="TextMain">
       <property name="text">
        <string>主相：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QPushButton" name="BtnExit">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>44</height>
        </size>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QCheckBox" name="BoxDirOnly">
       <property name="text">
        <string>只测磁旋</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
